.github/workflows/push.yml

```diff
@@ -1,33 +0,0 @@
----
-name: Tests
-
-on: 
-  push:
-  pull_request:
-  schedule:
-    - cron: '20 10 * * tue'
```

Comment:
the actions for `push` and `schedule` are missing from the new workflow

Reply:
Added them back to molecule.yml and ansible-lint.yml.